README.md

```diff
@@ -27,18 +23,16 @@
         <img src="https://img.shields.io/discord/516977525906341928?label=discord&color=informational&logo=Discord&logoColor=FFFFFF" alt="discord">
     </a>
     <!-- downloads -->
-    <a href="https://github.com/Moulberry/NotEnoughUpdates/releases" target="_blank">
```

Comment:
Do you really want to keep this?

Reply:
yes, I think we should update it to neu/neu once we make a release

Reply:
why dont we do it now

Reply:
ig, I just don't want to be sending notifications to people that are following the repo saying that there is a new release when there really isn't

Reply:
but we dont do releases here either?